testgrid/config/config.yaml

```diff
@@ -1781,6 +1781,10 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/pr-logs/directory/pull-cluster-registry-bazel
   days_of_results: 1
   num_columns_recent: 20
+# Istio
+- name: istio-api-postsubmit
```

Comment:
What is istio-api-postsubmit ?

Reply:
http://gcsweb.k8s.io/gcs/istio-prow/api-postsubmit/

Reply:
We want to track failure in istio/istio postsubmit: https://github.com/istio/test-infra/blob/master/prow/config.yaml#L1345

There are three run_after_success jobs: 
`e2e-suite-rbac-no_auth`
`e2e-suite-rbac-auth`
`e2e-cluster_wide-auth`

We want track failures from `istio-postsubmit`, `e2e-suite-rbac-no_auth` or `e2e-suite-rbac-auth`
but not `e2e-cluster_wide-auth` since it's flaky now.

Reply:
it also should be fine to add horribly failing jobs and just not alert on them :-)

Reply:
OK, so I guess my question would be what should we do about the job has "run_after_success". Do we just need to put parent job or we need to put every job (parent, and successor)?

Reply:
yeah you'll put every job, testgrid does not have the `run_after_success` knowledge and each of them will become one job entry. You can better organize them by dashboard group though.